docs/script.md

```diff
@@ -50,6 +50,9 @@ assert(script.toString() === 'OP_2 33 0x022df8750480ad5b26950b25c7ba79d3e37d75f6
 ```
 
 ### Pay to Script Hash (p2sh)
+
+DEPRECATED - Do not use p2sh on Bitcoin SV.
```

Comment:
Maybe note that standardness will be dropped with Genesis making it the preferred way of using special scripts?